install_config/configuring_form_based_authentication.adoc

```diff
@@ -0,0 +1,103 @@
+= Configuring Form-Based Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon xref:../install_config/ssd_for_ldap_failover.adoc[Setting
+up SSSD for LDAP Failover] and describes how to set up form-based authentication
+for signing into the {product-title} web console.
+
```

Comment:
I think it's worth including a brief description of what form-based auth is, and what benefit there would be to implementing it in an OpenShift cluster.